libsql-server/src/error.rs

```diff
@@ -103,6 +103,9 @@ pub enum Error {
     ProstDecode(#[from] prost::DecodeError),
     #[error("Shared schema error: {0}")]
     SharedSchemaCreationError(String),
+
+    #[error("migration error: {0}")]
```

Comment:
nit: It's probably more clear to say "schema migration" instead of just "migration".